content/docs/about/policies/_index.md

```diff
@@ -37,17 +37,68 @@ Review the policies that you can configure in kgateway and the level at which yo
+* Resources that target or serve other resources, such as Gateway > ListenerSet > HTTPRoute > Route rule.
+* Routes that are delegated, such as Parent HTTPRoute A > Child HTTPRoute B > Grandchild HTTPRoute C.
 
-For an example, see the [Policy inheritance](/docs/traffic-management/route-delegation/policy-inheritance/) guide.
+Policy merging applies to the following policies:
 
---> 
+* Native Kubernetes Gateway API policies, such as rewrites, timeouts, or retries.
+* {{< reuse "/docs/snippets/kgateway-capital.md" >}} TrafficPolicy.
+
+Resources that are higher in the parent-child hierarchy can use a special annotation to define how child resources inherit policies. This way, parent resources such as a Gateway or HTTPRoute can decide whether child resources can override the parent policies or not.
+
+### Merging annotation {#merging-annotation}
+
+The annotation on the parent resource is: `kgateway.dev/inherited-policy-priority`.
+
+The annotation takes four values:
+
+- `ShallowMergePreferChild` (default): Child policies take precedence over parent policies and the policies are shallow merged.
+- `ShallowMergePreferParent`: Parent policies take precedence over child policies and the policies are shallow merged.
+- `DeepMergePreferChild`: Child policies take precedence over parent policies and the policies are deep merged.
+- `DeepMergePreferParent`: Parent policies take precedence over child policies and the policies are deep merged.
+
+### Shallow or deep merging {#shallow-deep-merging}
+
+Merging ensures that policies from parent and child resources are combined without conflicts, using either _shallow_ or _deep_ strategies.
+
+**Shallow merging** means that the policies are merged at the top level. Only the top-level fields of the policies are considered for merging. If a field is present in both parent and child policies, the value from the higher priority policy is used. Priority is typically determined by specificity and creation time. The more specific (such as HTTPRoute rule over all the routes in the HTTPRoute) and older (created-first) policy takes precedence. Consider the following shallow merge scenario:
+
+* Parent policy adds a `x-season=summer` header.
+* Child policy adds `x-season=winter` and `x-holiday=christmas` headers.
+* Merging annotation is the default value, `ShallowMergePreferChild`.
+
+Resulting merged policy: The parent's `x-season` header is not included in the merged policy because the strategy is `ShallowMergePreferChild`.
+
+| Header | Value | Source |
+| -- | -- | -- |
+| `x-season` | `winter` | Child |
+| `x-holiday` | `christmas` | Child |
+
+**Deep merging** means that values from both parent and child policies can be combined. Currently, only [Transformation rules of a TrafficPolicy](/docs/traffic-management/transformations) can be deep merged. Consider the following deep merge scenario:
```

Comment:
same thing for traffic policy. also change link to ../ relative if we want to reuse (ignore these if not)